docs/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Version 3.33.0-eisop1 (April ?, 2023)
 
 **User-visible changes:**
 
+The format of the arguments of an `ObjectCreationNode` is correct when the node refers to an
+anonymous constructor with an explicit enclosing expression.
```

Comment:
```suggestion
Corrected the arguments to an `ObjectCreationNode` when the node refers to an
anonymous constructor invocation with an explicit enclosing expression.
```

Reply:
This should move down to `Implementation details`.

Reply:
Thanks! Done in my local.